homeassistant/components/evohome/coordinator.py

```diff
@@ -107,7 +106,7 @@ async def call_client_api(
             return None
 
         if update_state:  # wait a moment for system to quiesce before updating state
-            async_call_later(self.hass, 1, self._update_v2_api_state)
+            await self.hass.data[DOMAIN]["coordinator"].async_request_refresh()
```

Comment:
This coordinator design isn't good. Either the broker should be the coordinator and inherit `DataUpdateCoordinator` or the broker shouldn't be aware of the coordinator at all.

Reply:
I will sort this out, will wait for #123129 to get merged first (I can only do one thing at once).

Reply:
Thanks!